src/js/configurations/geography_sa.js

```diff
@@ -155,6 +155,9 @@ export class Config {
 
         const defaultConfig = {
             colors: d3schemeBlues[5],
+            positive_color_range: ["#eff3ff", "#08519c"],
+            negative_color_range: ["#fef0d9", "#b30000"],
```

Comment:
you fell into the same trap as me

here the right-hand item is the more intense colour, but you're using item 0 in the array as the most intense one when there are both positive and negative values.

Reply:
so I think the way it's used at https://github.com/OpenUpSA/wazimap-ng-ui/pull/665/files#diff-b2dbf83a3b8c7fe11dcab6b7b935a18c36ffbf277c9e6dbff75a0324de0c03dcR140 is fine - we could document that as "negative max", but then make the defaults be the right order: item 0 negative max, item 1 negative minimum